src/v/datalake/coordinator/coordinator.cc

```diff
@@ -494,6 +494,7 @@ coordinator::sync_add_files(
     }
 
     auto added_last_offset = entries.back().last_offset;
+    auto added_last_timestamp = entries.back().last_timestamp;
```

Comment:
>help with reconciliation.

Can you elaborate? And maybe add some comments in the corresponding state machines on how they are expected to be used

Reply:
dropped this bit of state from the coordinator, so it lives in the translation_stm only